.github/workflows/cd.yaml

```
@@ -0,0 +1,96 @@
+
+      - name: Gradle 캐시
+        uses: gradle/actions/setup-gradle@v3
+
+      - name: Grant execute permission
+        run: chmod +x gradlew
+
+      - name: Build JAR
+        run: ./gradlew clean bootJar
+
+      - name: Configure AWS Credentials
+        uses: aws-actions/configure-aws-credentials@v2
```

actionlint (1.7.12)

[error] the runner of "aws-actions/configure-aws-credentials@v2" action is too old to run on GitHub Actions. update the action's version to fix this issue

(action)